p2p_seednode.py intermittent issue: Expected messages "["Couldn't connect to peers from addrman after 10 seconds. Adding seednode (0.0.0.2) to addrfetch"]" does not partially match log:

https://cirrus-ci.com/task/6359355052785664?logs=ci#L0

```
315/315 - [1mp2p_seednode.py[0m failed, Duration: 604 s
[16:31:19.298] 
[16:31:19.298] [1mstdout:
[16:31:19.298] [0m2024-10-12T16:21:15.331000Z TestFramework (INFO): PRNG seed is: 5946564122551860139
[16:31:19.298] 2024-10-12T16:21:15.337000Z TestFramework (INFO): Initializing test directory /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0
[16:31:19.298] 2024-10-12T16:31:16.422000Z TestFramework (ERROR): Assertion failed
[16:31:19.298] Traceback (most recent call last):
[16:31:19.298]   File "/ci_container_base/test/functional/test_framework/test_framework.py", line 132, in main
[16:31:19.298]     self.run_test()
[16:31:19.298]   File "/ci_container_base/ci/scratch/build-x86_64-pc-linux-gnu/test/functional/p2p_seednode.py", line 50, in run_test
[16:31:19.298]     self.test_seednode_addrman_unreachable_peers()
[16:31:19.298]   File "/ci_container_base/ci/scratch/build-x86_64-pc-linux-gnu/test/functional/p2p_seednode.py", line 43, in test_seednode_addrman_unreachable_peers
[16:31:19.298]     with node.assert_debug_log(expected_msgs=[f"Couldn't connect to peers from addrman after {ADD_NEXT_SEEDNODE} seconds. Adding seednode ({seed_node}) to addrfetch"], unexpected_msgs=["Empty addrman, adding seednode"], timeout=ADD_NEXT_SEEDNODE * 1.5):
[16:31:19.298]   File "/usr/lib/python3.10/contextlib.py", line 142, in __exit__
[16:31:19.298]     next(self.gen)
[16:31:19.298]   File "/ci_container_base/test/functional/test_framework/test_node.py", line 519, in assert_debug_log
[16:31:19.298]     self._raise_assertion_error('Expected messages "{}" does not partially match log:\n\n{}\n\n'.format(str(expected_msgs), print_log))
[16:31:19.298]   File "/ci_container_base/test/functional/test_framework/test_node.py", line 197, in _raise_assertion_error
[16:31:19.298]     raise AssertionError(self._node_msg(msg))
[16:31:19.298] AssertionError: [node 0] Expected messages "["Couldn't connect to peers from addrman after 10 seconds. Adding seednode (0.0.0.2) to addrfetch"]" does not partially match log:
[16:31:19.298] 
[16:31:19.298]  - 2024-10-12T16:21:16.393038Z [http] [src/httpserver.cpp:307] [http_request_cb] [http] Received a POST request for / from 127.0.0.1:35668
[16:31:19.298]  - 2024-10-12T16:21:16.393073Z [httpworker.0] [src/rpc/request.cpp:232] [parse] [rpc] ThreadRPCServer method=stop user=__cookie__
[16:31:19.298]  - 2024-10-12T16:21:16.393134Z [init] [src/httpserver.cpp:510] [InterruptHTTPServer] [http] Interrupting HTTP server
[16:31:19.298]  - 2024-10-12T16:21:16.393189Z [init] [src/httprpc.cpp:379] [InterruptHTTPRPC] [rpc] Interrupting HTTP RPC server
[16:31:19.298]  - 2024-10-12T16:21:16.393197Z [init] [src/rpc/server.cpp:291] [operator()] [rpc] Interrupting RPC
[16:31:19.298]  - 2024-10-12T16:21:16.393217Z [init] [src/init.cpp:284] [Shutdown] Shutdown: In progress...
[16:31:19.298]  - 2024-10-12T16:21:16.393224Z [shutoff] [src/httprpc.cpp:384] [StopHTTPRPC] [rpc] Stopping HTTP RPC server
[16:31:19.298]  - 2024-10-12T16:21:16.393241Z [shutoff] [src/httpserver.cpp:772] [UnregisterHTTPHandler] [http] Unregistering HTTP handler for / (exactmatch 1)
[16:31:19.298]  - 2024-10-12T16:21:16.393248Z [shutoff] [src/httpserver.cpp:772] [UnregisterHTTPHandler] [http] Unregistering HTTP handler for /wallet/ (exactmatch 0)
[16:31:19.298]  - 2024-10-12T16:21:16.393258Z [shutoff] [src/rpc/server.cpp:303] [operator()] [rpc] Stopping RPC
[16:31:19.298]  - 2024-10-12T16:21:16.393316Z [shutoff] [src/rpc/server.cpp:306] [operator()] [rpc] RPC stopped.
[16:31:19.298]  - 2024-10-12T16:21:16.393324Z [shutoff] [src/httpserver.cpp:522] [StopHTTPServer] [http] Stopping HTTP server
[16:31:19.298]  - 2024-10-12T16:21:16.393328Z [shutoff] [src/httpserver.cpp:524] [StopHTTPServer] [http] Waiting for HTTP worker threads to exit
[16:31:19.298]  - 2024-10-12T16:21:16.393346Z [addcon] [src/util/thread.cpp:22] [TraceThread] addcon thread exit
[16:31:19.298]  - 2024-10-12T16:21:16.393467Z [shutoff] [src/httpserver.cpp:552] [StopHTTPServer] [http] Waiting for HTTP event thread to exit
[16:31:19.298]  - 2024-10-12T16:21:16.393488Z [http] [src/httpserver.cpp:357] [ThreadHTTP] [http] Exited http event loop
[16:31:19.298]  - 2024-10-12T16:21:16.393544Z [shutoff] [src/httpserver.cpp:558] [StopHTTPServer] [http] Stopped HTTP server
[16:31:19.298]  - 2024-10-12T16:21:16.396441Z [msghand] [src/util/thread.cpp:22] [TraceThread] msghand thread exit
[16:31:19.298]  - 2024-10-12T16:21:16.442060Z [net] [src/util/thread.cpp:22] [TraceThread] net thread exit
[16:31:19.298]  - 2024-10-12T16:21:21.299994Z [opencon] [src/netbase.cpp:589] [ConnectToSocket] [net] connection attempt to 0.0.0.1:18444 timed out
[16:31:19.298]  - 2024-10-12T16:21:21.300043Z [opencon] [src/util/thread.cpp:22] [TraceThread] opencon thread exit
[16:31:19.298]  - 2024-10-12T16:21:21.300459Z [shutoff] [src/net.cpp:2330] [DumpAddresses] [net] Flushed 10 addresses to peers.dat  0ms
[16:31:19.298]  - 2024-10-12T16:21:21.300477Z [shutoff] [src/logging/timer.h:56] [Log] DumpAnchors: Flush 0 outbound block-relay-only peer addresses to anchors.dat started
[16:31:19.298]  - 2024-10-12T16:21:21.300552Z [shutoff] [src/logging/timer.h:56] [Log] DumpAnchors: Flush 0 outbound block-relay-only peer addresses to anchors.dat completed (0.00s)
[16:31:19.298]  - 2024-10-12T16:21:21.300621Z [scheduler] [src/util/thread.cpp:22] [TraceThread] scheduler thread exit
[16:31:19.298]  - 2024-10-12T16:21:21.300889Z [shutoff] [src/node/mempool_persist.cpp:189] [DumpMempool] Writing 0 mempool transactions to file...
[16:31:19.298]  - 2024-10-12T16:21:21.300899Z [shutoff] [src/node/mempool_persist.cpp:199] [DumpMempool] Writing 0 unbroadcast transactions to file.
[16:31:19.298]  - 2024-10-12T16:21:21.300987Z [shutoff] [src/node/mempool_persist.cpp:210] [DumpMempool] Dumped mempool: 0.000s to copy, 0.000s to dump, 27 bytes dumped to file
[16:31:19.298]  - 2024-10-12T16:21:21.301000Z [shutoff] [src/policy/fees.cpp:1058] [FlushUnconfirmed] [estimatefee] Recorded 0 unconfirmed txs from mempool in 0.000s
[16:31:19.298]  - 2024-10-12T16:21:21.302555Z [shutoff] [src/policy/fees.cpp:956] [FlushFeeEstimates] Flushed fee estimates to fee_estimates.dat.
[16:31:19.298]  - 2024-10-12T16:21:21.302632Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write block and undo data to disk started
[16:31:19.298]  - 2024-10-12T16:21:21.302687Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write block and undo data to disk completed (0.04ms)
[16:31:19.298]  - 2024-10-12T16:21:21.302694Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write block index to disk started
[16:31:19.298]  - 2024-10-12T16:21:21.302723Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write block index to disk completed (0.02ms)
[16:31:19.298]  - 2024-10-12T16:21:21.302731Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write coins cache to disk (0 coins, 262kB) started
[16:31:19.298]  - 2024-10-12T16:21:21.302755Z [shutoff] [src/txdb.cpp:146] [BatchWrite] [coindb] Writing final batch of 0.00 MiB
[16:31:19.298]  - 2024-10-12T16:21:21.302772Z [shutoff] [src/txdb.cpp:148] [BatchWrite] [coindb] Committed 0 changed transaction outputs (out of 0) to coin database...
[16:31:19.298]  - 2024-10-12T16:21:21.302786Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write coins cache to disk (0 coins, 262kB) completed (0.05ms)
[16:31:19.298]  - 2024-10-12T16:21:21.302798Z [shutoff] [src/validationinterface.cpp:244] [ChainStateFlushed] [validation] Enqueuing ChainStateFlushed: block hash=51b0d1c56f5e26e55d80dc660b4aaf15519c96b500bef8fb30a7d5958d8de26e
[16:31:19.298]  - 2024-10-12T16:21:21.302815Z [shutoff] [src/validationinterface.cpp:244] [operator()] [validation] ChainStateFlushed: block hash=51b0d1c56f5e26e55d80dc660b4aaf15519c96b500bef8fb30a7d5958d8de26e
[16:31:19.298]  - 2024-10-12T16:21:21.302829Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write block and undo data to disk started
[16:31:19.298]  - 2024-10-12T16:21:21.302869Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write block and undo data to disk completed (0.03ms)
[16:31:19.298]  - 2024-10-12T16:21:21.302875Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write block index to disk started
[16:31:19.298]  - 2024-10-12T16:21:21.302983Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write block index to disk completed (0.01ms)
[16:31:19.298]  - 2024-10-12T16:21:21.302993Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write coins cache to disk (0 coins, 262kB) started
[16:31:19.298]  - 2024-10-12T16:21:21.303008Z [shutoff] [src/txdb.cpp:146] [BatchWrite] [coindb] Writing final batch of 0.00 MiB
[16:31:19.298]  - 2024-10-12T16:21:21.303018Z [shutoff] [src/txdb.cpp:148] [BatchWrite] [coindb] Committed 0 changed transaction outputs (out of 0) to coin database...
[16:31:19.298]  - 2024-10-12T16:21:21.303029Z [shutoff] [src/logging/timer.h:58] [Log] [bench] FlushStateToDisk: write coins cache to disk (0 coins, 262kB) completed (0.03ms)
[16:31:19.298]  - 2024-10-12T16:21:21.303037Z [shutoff] [src/validationinterface.cpp:244] [ChainStateFlushed] [validation] Enqueuing ChainStateFlushed: block hash=51b0d1c56f5e26e55d80dc660b4aaf15519c96b500bef8fb30a7d5958d8de26e
[16:31:19.298]  - 2024-10-12T16:21:21.305529Z [shutoff] [src/init.cpp:398] [Shutdown] Shutdown: done
[16:31:19.298]  - 
[16:31:19.298]  - 
[16:31:19.298]  - 
[16:31:19.298]  - 
[16:31:19.298]  - 
[16:31:19.298]  - 2024-10-12T16:21:21.315469Z [init] [src/init/common.cpp:149] [LogPackageVersion] Bitcoin Core version v28.99.0-87eae3ab1cb2-dirty (debug build)
[16:31:19.298]  - 2024-10-12T16:21:21.315478Z [init] [src/init.cpp:721] [InitParameterInteraction] parameter interaction: -bind set -> setting -listen=1
[16:31:19.298]  - 2024-10-12T16:21:21.319983Z [init] [src/node/chainstatemanager_args.cpp:64] [ApplyArgsManOptions] Script verification uses 2 additional threads
[16:31:19.298]  - 2024-10-12T16:21:21.320026Z [init] [src/kernel/context.cpp:20] [operator()] Using the 'x86_shani(1way,2way)' SHA256 implementation
[16:31:19.298]  - 2024-10-12T16:21:21.420768Z [init] [src/random.cpp:114] [ReportHardwareRand] Using RdSeed as an additional entropy source
[16:31:19.298]  - 2024-10-12T16:21:21.420770Z [init] [src/random.cpp:117] [ReportHardwareRand] Using RdRand as an additional entropy source
[16:31:19.298]  - 2024-10-12T16:21:21.429678Z [init] [src/init/common.cpp:120] [StartLogging] Default data directory /root/.bitcoin
[16:31:19.298]  - 2024-10-12T16:21:21.429684Z [init] [src/init/common.cpp:121] [StartLogging] Using data directory /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0/node0/regtest
[16:31:19.298]  - 2024-10-12T16:21:21.429695Z [init] [src/init/common.cpp:126] [StartLogging] Config file: /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0/node0/bitcoin.conf
[16:31:19.298]  - 2024-10-12T16:21:21.429718Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: regtest="1"
[16:31:19.298]  - 2024-10-12T16:21:21.429732Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] bind="127.0.0.1"
[16:31:19.298]  - 2024-10-12T16:21:21.429745Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] deprecatedrpc="create_bdb"
[16:31:19.298]  - 2024-10-12T16:21:21.429753Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] discover="0"
[16:31:19.298]  - 2024-10-12T16:21:21.429759Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] dnsseed="0"
[16:31:19.298]  - 2024-10-12T16:21:21.429764Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] fallbackfee="0.0002"
[16:31:19.298]  - 2024-10-12T16:21:21.429769Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] fixedseeds="0"
[16:31:19.298]  - 2024-10-12T16:21:21.429775Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] keypool="1"
[16:31:19.298]  - 2024-10-12T16:21:21.429779Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] listenonion="0"
[16:31:19.298]  - 2024-10-12T16:21:21.429790Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] natpmp="0"
[16:31:19.298]  - 2024-10-12T16:21:21.429796Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] peertimeout="999999999"
[16:31:19.298]  - 2024-10-12T16:21:21.429801Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] port="14000"
[16:31:19.298]  - 2024-10-12T16:21:21.429808Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] printtoconsole="0"
[16:31:19.298]  - 2024-10-12T16:21:21.429816Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] rpcdoccheck="1"
[16:31:19.298]  - 2024-10-12T16:21:21.429822Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] rpcport="19000"
[16:31:19.298]  - 2024-10-12T16:21:21.429829Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] rpcservertimeout="99000"
[16:31:19.298]  - 2024-10-12T16:21:21.429835Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] server="1"
[16:31:19.298]  - 2024-10-12T16:21:21.429841Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] shrinkdebugfile="0"
[16:31:19.298]  - 2024-10-12T16:21:21.429846Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] unsafesqlitesync="1"
[16:31:19.298]  - 2024-10-12T16:21:21.429853Z [init] [src/common/args.cpp:849] [logArgsPrefix] Config file arg: [regtest] upnp="0"
[16:31:19.298]  - 2024-10-12T16:21:21.429860Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: datadir="/ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0/node0"
[16:31:19.298]  - 2024-10-12T16:21:21.429867Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: debug=""
[16:31:19.298]  - 2024-10-12T16:21:21.429872Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: debugexclude="libevent"
[16:31:19.298]  - 2024-10-12T16:21:21.429877Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: debugexclude="leveldb"
[16:31:19.298]  - 2024-10-12T16:21:21.429883Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: debugexclude="rand"
[16:31:19.298]  - 2024-10-12T16:21:21.429888Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: disablewallet=""
[16:31:19.298]  - 2024-10-12T16:21:21.429894Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: loglevel="trace"
[16:31:19.298]  - 2024-10-12T16:21:21.429900Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: logsourcelocations=""
[16:31:19.298]  - 2024-10-12T16:21:21.429905Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: logthreadnames=""
[16:31:19.298]  - 2024-10-12T16:21:21.429910Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: logtimemicros=""
[16:31:19.298]  - 2024-10-12T16:21:21.429915Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: seednode="0.0.0.2"
[16:31:19.298]  - 2024-10-12T16:21:21.429921Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: uacomment="testnode0"
[16:31:19.298]  - 2024-10-12T16:21:21.429926Z [init] [src/common/args.cpp:849] [logArgsPrefix] Command-line arg: v2transport="0"
[16:31:19.298]  - 2024-10-12T16:21:21.429931Z [init] [src/init.cpp:1294] [AppInitMain] Using at most 125 automatic connections (1048576 file descriptors available)
[16:31:19.298]  - 2024-10-12T16:21:21.430024Z [init] [src/wallet/init.cpp:129] [Construct] Wallet disabled!
[16:31:19.298]  - 2024-10-12T16:21:21.434263Z [scheduler] [src/util/thread.cpp:20] [TraceThread] scheduler thread start
[16:31:19.298]  - 2024-10-12T16:21:21.440309Z [init] [src/httpserver.cpp:241] [InitHTTPAllowList] [http] Allowing HTTP connections from: 127.0.0.0/8 ::1/128 
[16:31:19.298]  - 2024-10-12T16:21:21.440370Z [init] [src/httpserver.cpp:390] [HTTPBindAddresses] Binding RPC on address ::1 port 19000
[16:31:19.298]  - 2024-10-12T16:21:21.440474Z [init] [src/httpserver.cpp:390] [HTTPBindAddresses] Binding RPC on address 127.0.0.1 port 19000
[16:31:19.298]  - 2024-10-12T16:21:21.440533Z [init] [src/httpserver.cpp:475] [InitHTTPServer] [http] Initialized HTTP server
[16:31:19.298]  - 2024-10-12T16:21:21.440544Z [init] [src/httpserver.cpp:477] [InitHTTPServer] [http] creating work queue of depth 16
[16:31:19.298]  - 2024-10-12T16:21:21.440550Z [init] [src/rpc/server.cpp:282] [StartRPC] [rpc] Starting RPC
[16:31:19.298]  - 2024-10-12T16:21:21.440555Z [init] [src/httprpc.cpp:361] [StartHTTPRPC] [rpc] Starting HTTP RPC server
[16:31:19.298]  - 2024-10-12T16:21:21.440562Z [init] [src/httprpc.cpp:297] [InitRPCAuthentication] Using random cookie authentication.
[16:31:19.298]  - 2024-10-12T16:21:21.440669Z [init] [src/rpc/request.cpp:132] [GenerateAuthCookie] Generated RPC authentication cookie /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0/node0/regtest/.cookie
[16:31:19.298]  - 2024-10-12T16:21:21.440685Z [init] [src/rpc/request.cpp:133] [GenerateAuthCookie] Permissions used for cookie: rw-------
[16:31:19.298]  - 2024-10-12T16:21:21.440703Z [init] [src/httpserver.cpp:757] [RegisterHTTPHandler] [http] Registering HTTP handler for / (exactmatch 1)
[16:31:19.298]  - 2024-10-12T16:21:21.440716Z [init] [src/httpserver.cpp:757] [RegisterHTTPHandler] [http] Registering HTTP handler for /wallet/ (exactmatch 0)
[16:31:19.298]  - 2024-10-12T16:21:21.440732Z [init] [src/httpserver.cpp:500] [StartHTTPServer] Starting HTTP server with 4 worker threads
[16:31:19.298]  - 2024-10-12T16:21:21.441096Z [init] [src/init.cpp:1415] [AppInitMain] Using /16 prefix for IP bucketing
[16:31:19.298]  - 2024-10-12T16:21:21.441111Z [init] [src/noui.cpp:57] [noui_InitMessage] init message: Loading P2P addresses…
[16:31:19.298]  - 2024-10-12T16:21:21.441453Z [init] [src/logging/timer.h:58] [Log] [addrman] CheckAddrman: new 10, tried 0, total 10 started
[16:31:19.298]  - 2024-10-12T16:21:21.441575Z [init] [src/logging/timer.h:58] [Log] [addrman] CheckAddrman: completed (0.11ms)
[16:31:19.298]  - 2024-10-12T16:21:21.441587Z [init] [src/addrdb.cpp:202] [LoadAddrman] Loaded 10 addresses from peers.dat  0ms
[16:31:19.298]  - 2024-10-12T16:21:21.443693Z [init] [src/noui.cpp:57] [noui_InitMessage] init message: Loading banlist…
[16:31:19.298]  - 2024-10-12T16:21:21.443761Z [init] [src/banman.cpp:39] [LoadBanlist] [net] Loaded 0 banned node addresses/subnets  0ms
[16:31:19.298]  - 2024-10-12T16:21:21.443782Z [init] [src/net.cpp:2363] [SetTryNewOutboundPeer] [net] setting try another outbound peer=false
[16:31:19.298]  - 2024-10-12T16:21:21.443791Z [init] [src/net.cpp:3130] [SetNetworkActive] SetNetworkActive: true
[16:31:19.298]  - 2024-10-12T16:21:21.452076Z [http] [src/httpserver.cpp:354] [ThreadHTTP] [http] Entering http event loop
[16:31:19.298]  - 2024-10-12T16:21:21.455291Z [init] [src/policy/fees.cpp:469] [Read] [estimatefee] Reading estimates: 190 buckets counting confirms up to 48 blocks
[16:31:19.298]  - 2024-10-12T16:21:21.455523Z [init] [src/policy/fees.cpp:469] [Read] [estimatefee] Reading estimates: 190 buckets counting confirms up to 12 blocks
[16:31:19.298]  - 2024-10-12T16:21:21.456034Z [init] [src/policy/fees.cpp:469] [Read] [estimatefee] Reading estimates: 190 buckets counting confirms up to 1008 blocks
[16:31:19.298]  - 2024-10-12T16:21:21.464343Z [init] [src/init.cpp:1610] [AppInitMain] Cache configuration:
[16:31:19.298]  - 2024-10-12T16:21:21.464364Z [init] [src/init.cpp:1611] [AppInitMain] * Using 2.0 MiB for block index database
[16:31:19.298]  - 2024-10-12T16:21:21.464379Z [init] [src/init.cpp:1619] [AppInitMain] * Using 8.0 MiB for chain state database
[16:31:19.298]  - 2024-10-12T16:21:21.464443Z [init] [src/init.cpp:1196] [InitAndLoadChainstate] * Using 440.0 MiB for in-memory UTXO set (plus up to 286.1 MiB of unused mempool space)
[16:31:19.298]  - 2024-10-12T16:21:21.464517Z [init] [src/node/chainstatemanager_args.cpp:64] [ApplyArgsManOptions] Script verification uses 2 additional threads
[16:31:19.298]  - 2024-10-12T16:21:21.464723Z [init] [src/node/blockstorage.cpp:1176] [InitBlocksdirXorKey] Using obfuscation key for blocksdir *.dat files (/ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0/node0/regtest/blocks): '627e2c92ffe2480d'
[16:31:19.298]  - 2024-10-12T16:21:21.493864Z [init] [src/script/sigcache.cpp:35] [SignatureCache] Using 16 MiB out of 16 MiB requested for signature cache, able to store 524288 elements
[16:31:19.298]  - 2024-10-12T16:21:21.532382Z [init] [src/validation.cpp:2107] [ValidationCache] Using 16 MiB out of 16 MiB requested for script execution cache, able to store 524288 elements
[16:31:19.298]  - 2024-10-12T16:21:21.532452Z [init] [src/noui.cpp:57] [noui_InitMessage] init message: Loading block index…
[16:31:19.298]  - 2024-10-12T16:21:21.532466Z [init] [src/node/chainstate.cpp:168] [LoadChainstate] Validating signatures for all blocks.
[16:31:19.298]  - 2024-10-12T16:21:21.532476Z [init] [src/node/chainstate.cpp:170] [LoadChainstate] Setting nMinimumChainWork=0000000000000000000000000000000000000000000000000000000000000000
[16:31:19.298]  - 2024-10-12T16:21:21.532532Z [init] [src/dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0/node0/regtest/blocks/index
[16:31:19.298]  - 2024-10-12T16:21:21.533400Z [init] [src/dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully
[16:31:19.298]  - 2024-10-12T16:21:21.533423Z [init] [src/dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0/node0/regtest/blocks/index: 0000000000000000
[16:31:19.298]  - 2024-10-12T16:21:21.534597Z [init] [src/node/blockstorage.cpp:508] [LoadBlockIndexDB] LoadBlockIndexDB: last block file = 0
[16:31:19.298]  - 2024-10-12T16:21:21.534618Z [init] [src/node/blockstorage.cpp:512] [LoadBlockIndexDB] LoadBlockIndexDB: last block file info: CBlockFileInfo(blocks=201, size=52791, heights=0...200, time=2011-02-02...2024-10-12)
[16:31:19.298]  - 2024-10-12T16:21:21.534626Z [init] [src/node/blockstorage.cpp:523] [LoadBlockIndexDB] Checking all blk files are present...
[16:31:19.298]  - 2024-10-12T16:21:21.535509Z [init] [src/node/chainstate.cpp:108] [CompleteChainstateInitialization] Initializing chainstate Chainstate [ibd] @ height -1 (null)
[16:31:19.298]  - 2024-10-12T16:21:21.535558Z [init] [src/dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0/node0/regtest/chainstate
[16:31:19.298]  - 2024-10-12T16:21:21.551075Z [init] [src/dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully
[16:31:19.298]  - 2024-10-12T16:21:21.551118Z [init] [src/dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0/node0/regtest/chainstate: ad9335395de104ac
[16:31:19.298]  - 2024-10-12T16:21:21.551254Z [init] [src/validation.cpp:4667] [LoadChainTip] Loaded best chain: hashBestChain=51b0d1c56f5e26e55d80dc660b4aaf15519c96b500bef8fb30a7d5958d8de26e height=200 date=2024-10-12T16:21:15Z progress=1.000000
[16:31:19.298]  - 2024-10-12T16:21:21.551322Z [init] [src/dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0/node0/regtest/chainstate
[16:31:19.298]  - 2024-10-12T16:21:21.551604Z [init] [src/dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully
[16:31:19.298]  - 2024-10-12T16:21:21.551647Z [init] [src/dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20241012_160653/p2p_seednode_0/node0/regtest/chainstate: ad9335395de104ac
[16:31:19.298]  - 2024-10-12T16:21:21.551660Z [init] [src/validation.cpp:5535] [ResizeCoinsCaches] [Chainstate [ibd] @ height 200 (51b0d1c56f5e26e55d80dc660b4aaf15519c96b500bef8fb30a7d5958d8de26e)] resized coinsdb cache to 8.0 MiB
[16:31:19.298]  - 2024-10-12T16:21:21.551667Z [init] [src/validation.cpp:5537] [ResizeCoinsCaches] [Chainstate [ibd] @ height 200 (51b0d1c56f5e26e55d80dc660b4aaf15519c96b500bef8fb30a7d5958d8de26e)] resized coinstip cache to 440.0 MiB
[16:31:19.298]  - 2024-10-12T16:21:21.551689Z [init] [src/noui.cpp:57] [noui_InitMessage] init message: Verifying blocks…
[16:31:19.298]  - 2024-10-12T16:21:21.551724Z [init] [src/validation.cpp:4703] [VerifyDB] Verifying last 6 blocks at level 3
[16:31:19.298]  - 2024-10-12T16:21:21.551742Z [init] [src/validation.cpp:4712] [VerifyDB] Verification progress: 0%
[16:31:19.298]  - 2024-10-12T16:21:21.551858Z [init] [src/validation.cpp:4720] [VerifyDB] Verification progress: 16%
[16:31:19.298]  - 2024-10-12T16:21:21.551929Z [init] [src/validation.cpp:4720] [VerifyDB] Verification progress: 33%
[16:31:19.298]  - 2024-10-12T16:21:21.552558Z [init] [src/validation.cpp:4720] [VerifyDB] Verification progress: 50%
[16:31:19.298]  - 2024-10-12T16:21:21.552628Z [init] [src/validation.cpp:4720] [VerifyDB] Verification progress: 66%
[16:31:19.298]  - 2024-10-12T16:21:21.552688Z [init] [src/validation.cpp:4720] [VerifyDB] Verification progress: 83%
[16:31:19.298]  - 2024-10-12T16:21:21.552746Z [init] [src/validation.cpp:4720] [VerifyDB] Verification progress: 99%
[16:31:19.298]  - 2024-10-12T16:21:21.552754Z [init] [src/validation.cpp:4814] [VerifyDB] Verification: No coin database inconsistencies in last 6 blocks (6 transactions)
[16:31:19.298]  - 2024-10-12T16:21:21.552775Z [init] [src/init.cpp:1268] [InitAndLoadChainstate]  block index              20ms
[16:31:19.298]  - 2024-10-12T16:21:21.552820Z [init] [src/init.cpp:1718] [AppInitMain] Setting NODE_NETWORK on non-prune mode
[16:31:19.298]  - 2024-10-12T16:21:21.552895Z [init] [src/init.cpp:1818] [AppInitMain] block tree size = 201
[16:31:19.298]  - 2024-10-12T16:21:21.552903Z [init] [src/init.cpp:1831] [AppInitMain] nBestHeight = 200
[16:31:19.298]  - 2024-10-12T16:21:21.552982Z [init] [src/net.cpp:3103] [BindListenPort] Bound to 127.0.0.1:14000
[16:31:19.298]  - 2024-10-12T16:21:21.553018Z [init] [src/addrdb.cpp:234] [ReadAnchors] Loaded 0 addresses from "anchors.dat"
[16:31:19.298]  - 2024-10-12T16:21:21.553052Z [init] [src/net.cpp:3259] [Start] 0 block-relay-only anchors will be tried for connections.
[16:31:19.298]  - 2024-10-12T16:21:21.553061Z [init] [src/noui.cpp:57] [noui_InitMessage] init message: Starting network threads…
[16:31:19.298]  - 2024-10-12T16:21:21.553101Z [init] [src/net.cpp:3293] [Start] DNS seeding disabled
[16:31:19.298]  - 2024-10-12T16:21:21.553194Z [init] [src/noui.cpp:57] [noui_InitMessage] init message: Done loading
[16:31:19.298]  - 2024-10-12T16:21:21.553230Z [init] [src/logging/timer.h:58] [Log] [lock] Enter: lock contention newTaskMutex, ./src/scheduler.cpp:74 started
[16:31:19.298]  - 2024-10-12T16:21:21.553244Z [init] [src/logging/timer.h:58] [Log] [lock] Enter: lock contention newTaskMutex, ./src/scheduler.cpp:74 completed (1μs)
[16:31:19.298]  - 2024-10-12T16:21:21.556945Z [initload] [src/util/thread.cpp:20] [TraceThread] initload thread start
[16:31:19.298]  - 2024-10-12T16:21:21.557006Z [initload] [src/node/mempool_persist.cpp:73] [LoadMempool] Loading 0 mempool transactions from file...
[16:31:19.298]  - 2024-10-12T16:21:21.557015Z [initload] [src/node/mempool_persist.cpp:145] [LoadMempool] Imported mempool transactions from file: 0 succeeded, 0 failed, 0 expired, 0 already there, 0 waiting for initial broadcast
[16:31:19.298]  - 2024-10-12T16:21:21.557023Z [initload] [src/util/thread.cpp:22] [TraceThread] initload thread exit
[16:31:19.298]  - 2024-10-12T16:21:21.557554Z [net] [src/util/thread.cpp:20] [TraceThread] net thread start
[16:31:19.298]  - 2024-10-12T16:21:21.563475Z [http] [src/httpserver.cpp:307] [http_request_cb] [http] Received a POST request for / from 127.0.0.1:42764
[16:31:19.298]  - 2024-10-12T16:21:21.563543Z [httpworker.3] [src/rpc/request.cpp:232] [parse] [rpc] ThreadRPCServer method=getblockcount user=__cookie__
[16:31:19.298]  - 2024-10-12T16:21:21.564328Z [http] [src/httpserver.cpp:307] [http_request_cb] [http] Received a POST request for / from 127.0.0.1:42764
[16:31:19.298]  - 2024-10-12T16:21:21.564371Z [httpworker.2] [src/rpc/request.cpp:232] [parse] [rpc] ThreadRPCServer method=getmempoolinfo user=__cookie__
[16:31:19.298]  - 2024-10-12T16:21:21.565038Z [http] [src/httpserver.cpp:307] [http_request_cb] [http] Received a POST request for / from 127.0.0.1:42764
[16:31:19.298]  - 2024-10-12T16:21:21.565081Z [httpworker.1] [src/rpc/request.cpp:232] [parse] [rpc] ThreadRPCServer method=setmocktime user=__cookie__
[16:31:19.298]  - 2024-10-12T16:21:21.566720Z (mocktime: 2024-10-12T16:21:32Z) [addcon] [src/util/thread.cpp:20] [TraceThread] addcon thread start
[16:31:19.298]  - 2024-10-12T16:21:21.566815Z (mocktime: 2024-10-12T16:21:32Z) [msghand] [src/util/thread.cpp:20] [TraceThread] msghand thread start
[16:31:19.298]  - 2024-10-12T16:21:21.566845Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/util/thread.cpp:20] [TraceThread] opencon thread start
[16:31:19.298]  - 2024-10-12T16:21:21.566893Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/net.cpp:2499] [ThreadOpenConnections] Fixed seeds are disabled
[16:31:19.298]  - 2024-10-12T16:21:22.068407Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 133.103.54.242:8338 from new
[16:31:19.298]  - 2024-10-12T16:21:22.068458Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/net.cpp:407] [ConnectNode] [net] trying v1 connection 133.103.54.242:8338 lastseen=0.0hrs
[16:31:19.298]  - 2024-10-12T16:21:27.072005Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/netbase.cpp:589] [ConnectToSocket] [net] connection attempt to 133.103.54.242:8338 timed out
[16:31:19.298]  - 2024-10-12T16:21:27.573253Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 161.156.7.7:8333 from new
[16:31:19.298]  - 2024-10-12T16:21:27.573299Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/net.cpp:407] [ConnectNode] [net] trying v1 connection 161.156.7.7:8333 lastseen=0.0hrs
[16:31:19.298]  - 2024-10-12T16:21:32.575996Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/netbase.cpp:589] [ConnectToSocket] [net] connection attempt to 161.156.7.7:8333 timed out
[16:31:19.298]  - 2024-10-12T16:21:33.077260Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 150.20.149.177:8342 from new
[16:31:19.298]  - 2024-10-12T16:21:33.077310Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/net.cpp:407] [ConnectNode] [net] trying v1 connection 150.20.149.177:8342 lastseen=0.0hrs
[16:31:19.298]  - 2024-10-12T16:21:38.079990Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/netbase.cpp:589] [ConnectToSocket] [net] connection attempt to 150.20.149.177:8342 timed out
[16:31:19.298]  - 2024-10-12T16:21:38.580385Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 166.12.77.71:8339 from new
[16:31:19.298]  - 2024-10-12T16:21:38.580424Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/net.cpp:407] [ConnectNode] [net] trying v1 connection 166.12.77.71:8339 lastseen=0.0hrs
[16:31:19.298]  - 2024-10-12T16:21:43.584073Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/netbase.cpp:589] [ConnectToSocket] [net] connection attempt to 166.12.77.71:8339 timed out
[16:31:19.298]  - 2024-10-12T16:21:44.086328Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 162.126.230.57:8334 from new
[16:31:19.298]  - 2024-10-12T16:21:44.086395Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/net.cpp:407] [ConnectNode] [net] trying v1 connection 162.126.230.57:8334 lastseen=0.0hrs
[16:31:19.298]  - 2024-10-12T16:21:49.091629Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/netbase.cpp:589] [ConnectToSocket] [net] connection attempt to 162.126.230.57:8334 timed out
[16:31:19.298]  - 2024-10-12T16:21:49.592885Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 166.136.83.164:8336 from new
[16:31:19.298]  - 2024-10-12T16:21:49.592976Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/net.cpp:407] [ConnectNode] [net] trying v1 connection 166.136.83.164:8336 lastseen=0.0hrs
[16:31:19.298]  - 2024-10-12T16:21:54.598200Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/netbase.cpp:589] [ConnectToSocket] [net] connection attempt to 166.136.83.164:8336 timed out
[16:31:19.298]  - 2024-10-12T16:21:55.098817Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 137.145.53.228:8335 from new
[16:31:19.298]  - 2024-10-12T16:21:55.098881Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/net.cpp:407] [ConnectNode] [net] trying v1 connection 137.145.53.228:8335 lastseen=0.0hrs
[16:31:19.298]  - 2024-10-12T16:21:55.260327Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/netbase.cpp:566] [LogConnectFailure] [net] connect() to 137.145.53.228:8335 failed after wait: Network is unreachable (101)
[16:31:19.298]  - 2024-10-12T16:21:55.762932Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 155.177.170.113:8337 from new
[16:31:19.298]  - 2024-10-12T16:21:55.762998Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/net.cpp:407] [ConnectNode] [net] trying v1 connection 155.177.170.113:8337 lastseen=0.0hrs
[16:31:19.298]  - 2024-10-12T16:22:00.768191Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/netbase.cpp:589] [ConnectToSocket] [net] connection attempt to 155.177.170.113:8337 timed out
[16:31:19.298]  - 2024-10-12T16:22:01.269125Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 145.196.149.12:8340 from new
[16:31:19.298]  - 2024-10-12T16:22:01.269193Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/net.cpp:407] [ConnectNode] [net] trying v1 connection 145.196.149.12:8340 lastseen=0.0hrs
[16:31:19.298]  - 2024-10-12T16:22:06.274410Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/netbase.cpp:589] [ConnectToSocket] [net] connection attempt to 145.196.149.12:8340 timed out
[16:31:19.298]  - 2024-10-12T16:22:06.553384Z (mocktime: 2024-10-12T16:21:32Z) [scheduler] [src/net.cpp:2368] [StartExtraBlockRelayPeers] [net] enabling extra block-relay-only peers
[16:31:19.298]  - 2024-10-12T16:22:06.778861Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 162.126.230.57:8334 from new
[16:31:19.298]  - 2024-10-12T16:22:06.786716Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 161.156.7.7:8333 from new
[16:31:19.298]  - 2024-10-12T16:22:06.792757Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 166.136.83.164:8336 from new
[16:31:19.298]  - 2024-10-12T16:22:06.792918Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 158.76.62.78:8341 from new
[16:31:19.298]  - 2024-10-12T16:22:06.792972Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/net.cpp:407] [ConnectNode] [net] trying v1 connection 158.76.62.78:8341 lastseen=0.0hrs
[16:31:19.298]  - 2024-10-12T16:22:11.798171Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/netbase.cpp:589] [ConnectToSocket] [net] connection attempt to 158.76.62.78:8341 timed out
[16:31:19.298]  - 2024-10-12T16:22:12.317420Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 161.156.7.7:8333 from new
[16:31:19.298]  - 2024-10-12T16:22:12.335329Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 158.76.62.78:8341 from new
[16:31:19.298]  - 2024-10-12T16:22:12.353269Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 133.103.54.242:8338 from new
[16:31:19.298]  - 2024-10-12T16:22:12.363669Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 155.177.170.113:8337 from new
[16:31:19.298]  - 2024-10-12T16:22:12.376513Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 166.12.77.71:8339 from new
[16:31:19.298]  - 2024-10-12T16:22:12.381922Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 133.103.54.242:8338 from new
[16:31:19.298]  - 2024-10-12T16:22:12.390055Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 133.103.54.242:8338 from new
[16:31:19.298]  - 2024-10-12T16:22:12.405798Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 150.20.149.177:8342 from new
[16:31:19.298]  - 2024-10-12T16:22:12.411326Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 145.196.149.12:8340 from new
[16:31:19.298]  - 2024-10-12T16:22:12.419576Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 137.145.53.228:8335 from new
[16:31:19.298]  - 2024-10-12T16:22:12.450449Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 145.196.149.12:8340 from new
[16:31:19.298]  - 2024-10-12T16:22:12.467137Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 161.156.7.7:8333 from new
[16:31:19.298]  - 2024-10-12T16:22:12.485809Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 133.103.54.242:8338 from new
[16:31:19.298]  - 2024-10-12T16:22:12.490678Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 158.76.62.78:8341 from new
[16:31:19.298]  - 2024-10-12T16:22:12.513438Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 166.12.77.71:8339 from new
[16:31:19.298]  - 2024-10-12T16:22:12.536593Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 166.136.83.164:8336 from new
[16:31:19.298]  - 2024-10-12T16:22:12.549660Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 150.20.149.177:8342 from new
[16:31:19.298]  - 2024-10-12T16:22:12.557612Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 158.76.62.78:8341 from new
[16:31:19.298]  - 2024-10-12T16:22:12.558180Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 162.126.230.57:8334 from new
[16:31:19.298]  - 2024-10-12T16:22:12.572447Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 166.136.83.164:8336 from new
[16:31:19.298]  - 2024-10-12T16:22:12.582945Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 166.136.83.164:8336 from new
[16:31:19.298]  - 2024-10-12T16:22:12.599610Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 161.156.7.7:8333 from new
[16:31:19.298]  - 2024-10-12T16:22:12.616925Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 155.177.170.113:8337 from new
[16:31:19.298]  - 2024-10-12T16:22:12.629822Z (mocktime: 2024-10-12T16:21:32Z) [opencon] [src/addrman.cpp:786] [Select_] [addrman] Selected 150.20.149.177:8342 from new

--- a/test/functional/p2p_seednode.py
+++ b/test/functional/p2p_seednode.py
@@ -20,17 +20,18 @@ class P2PSeedNodes(BitcoinTestFramework):
         self.disable_autoconnect = False
 
     def test_no_seednode(self):
-        # Check that if no seednode is provided, the node proceeds as usual (without waiting)
+        self.log.info("Check that if no seednode is provided, the node proceeds as usual (without waiting)")
         with self.nodes[0].assert_debug_log(expected_msgs=[], unexpected_msgs=["Empty addrman, adding seednode", f"Couldn't connect to peers from addrman after {ADD_NEXT_SEEDNODE} seconds. Adding seednode"], timeout=ADD_NEXT_SEEDNODE):
             self.restart_node(0)
 
     def test_seednode_empty_addrman(self):
         seed_node = "0.0.0.1"
-        # Check that the seednode is added to m_addr_fetches on bootstrap on an empty addrman
+        self.log.info("Check that the seednode is immediately added on bootstrap on an empty addrman")
         with self.nodes[0].assert_debug_log(expected_msgs=[f"Empty addrman, adding seednode ({seed_node}) to addrfetch"], timeout=ADD_NEXT_SEEDNODE):
             self.restart_node(0, extra_args=[f'-seednode={seed_node}'])
 
     def test_seednode_addrman_unreachable_peers(self):
+        self.log.info("Check that if addrman is non-empty, seednodes are queried with a delay")
         seed_node = "0.0.0.2"
         node = self.nodes[0]
         # Fill the addrman with unreachable nodes
@@ -40,8 +41,10 @@ class P2PSeedNodes(BitcoinTestFramework):
             node.addpeeraddress(ip, port)
 
         # Restart the node so seednode is processed again
-        with node.assert_debug_log(expected_msgs=[f"Couldn't connect to peers from addrman after {ADD_NEXT_SEEDNODE} seconds. Adding seednode ({seed_node}) to addrfetch"], unexpected_msgs=["Empty addrman, adding seednode"], timeout=ADD_NEXT_SEEDNODE * 1.5):
+        with node.assert_debug_log(expected_msgs=["trying v1 connection"], timeout=ADD_NEXT_SEEDNODE):
             self.restart_node(0, extra_args=[f'-seednode={seed_node}'])
+
+        with node.assert_debug_log(expected_msgs=[f"Couldn't connect to peers from addrman after {ADD_NEXT_SEEDNODE} seconds. Adding seednode ({seed_node}) to addrfetch"], unexpected_msgs=["Empty addrman, adding seednode"], timeout=ADD_NEXT_SEEDNODE * 1.5):
             node.setmocktime(int(time.time()) + ADD_NEXT_SEEDNODE + 1)
 
     def run_test(self):
@@ -52,4 +55,3 @@ class P2PSeedNodes(BitcoinTestFramework):
 
 if __name__ == '__main__':
     P2PSeedNodes(__file__).main()
-

--- a/test/functional/p2p_seednode.py
+++ b/test/functional/p2p_seednode.py
@@ -30,7 +30,7 @@ class P2PSeedNodes(BitcoinTestFramework):
         with self.nodes[0].assert_debug_log(expected_msgs=[f"Empty addrman, adding seednode ({seed_node}) to addrfetch"], timeout=ADD_NEXT_SEEDNODE):
             self.restart_node(0, extra_args=[f'-seednode={seed_node}'])
 
-    def test_seednode_addrman_unreachable_peers(self):
+    def test_seednode_non_empty_addrman(self):
         self.log.info("Check that if addrman is non-empty, seednodes are queried with a delay")
         seed_node = "0.0.0.2"
         node = self.nodes[0]
@@ -40,9 +40,9 @@ class P2PSeedNodes(BitcoinTestFramework):
             port = 8333 + i
             node.addpeeraddress(ip, port)
 
-        # Restart the node so seednode is processed again
+        # Restart the node so seednode is processed again. Specify a non-working proxy to make sure no actual connections to random IPs are attempted.
         with node.assert_debug_log(expected_msgs=["trying v1 connection"], timeout=ADD_NEXT_SEEDNODE):
-            self.restart_node(0, extra_args=[f'-seednode={seed_node}'])
+            self.restart_node(0, extra_args=[f'-seednode={seed_node}', '-proxy=127.0.0.1:1'])
 
         with node.assert_debug_log(expected_msgs=[f"Couldn't connect to peers from addrman after {ADD_NEXT_SEEDNODE} seconds. Adding seednode ({seed_node}) to addrfetch"], unexpected_msgs=["Empty addrman, adding seednode"], timeout=ADD_NEXT_SEEDNODE * 1.5):
             node.setmocktime(int(time.time()) + ADD_NEXT_SEEDNODE + 1)
@@ -50,7 +50,7 @@ class P2PSeedNodes(BitcoinTestFramework):
     def run_test(self):
         self.test_no_seednode()
         self.test_seednode_empty_addrman()
-        self.test_seednode_addrman_unreachable_peers()
+        self.test_seednode_non_empty_addrman()
 
 
 if __name__ == '__main__':

